tests/functional/spec/features/routing/and.spec.js

```diff
@@ -0,0 +1,38 @@
+import FirstNumberQuestionPage from "../../../generated_pages/new_routing_and/number-question-1.page";
+import SecondNumberQuestionPage from "../../../generated_pages/new_routing_and/number-question-2.page";
+import CorrectAnswerPage from "../../../generated_pages/new_routing_and/correct-answer.page";
+import IncorrectAnswerPage from "../../../generated_pages/new_routing_and/incorrect-answer.page";
+
+describe("Feature: Routing - And Operator", () => {
+  describe("Equals", () => {
+    describe("Given I start the and operator routing survey", () => {
+      beforeEach(() => {
+        browser.openQuestionnaire("test_new_routing_and.json");
+      });
+
+      it("When I enter both answers correctly, Then I should be routed to the correct page", () => {
```

Comment:
I am not a huge for of the generic `When`, `Then`, Explicit is better than implicit. I prefer if it was more like `When I enter 123 and 321, Then ...`. The outputs of the func test become useful then and can be replayed without looking, at the file for inputs. Maybe see if others agree. Although not a blocker, if these are being kept like this it might be worth rationalizing it, as it is almost identical in all 3 files. (this might change but on my comment about the `not` operator`)

Reply:
I've made them more specific now and the not tests have been updated.